pcap/net_parse.go

```diff
@@ -48,18 +48,27 @@ func (fact *tcpStreamFactory) New(netFlow, tcpFlow gopacket.Flow, _ *layers.TCP,
 	return newTCPStream(fact.clock, netFlow, fact.outChan, fact.fs)
 }
 
+type NetworkTrafficObserver func(gopacket.Packet)
```

Comment:
Please document.

Reply:
Fixed in https://github.com/akitasoftware/akita-cli/pull/18/commits/956c721b9bd0c25cc7d45c5583260cf4c5a70f3c